src/Versity.Users/External/Presentation/Controllers/ErrorController.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Application.Exceptions;
+using Microsoft.AspNetCore.Diagnostics;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Presentation.Controllers;
+
+public sealed class ErrorController : ControllerBase
+{
+    [ApiExplorerSettings(IgnoreApi = true)]
+    [Route("/error")]
+    public IActionResult HandleError()
+    {
+        var exception = HttpContext.Features.Get<IExceptionHandlerFeature>()?.Error;
+
+        if (exception is HttpResponseException httpResponseException)
+            return Problem(
+                title: httpResponseException.Value?.ToString(), 
+                statusCode: httpResponseException.StatusCode);
+
+        return Problem(title: "Ops 🤨! Something went wrong...");
```

Comment:
What is problem here?)

Reply:
At the moment, I only inform the user about errors if the Exception was of my type (ExceptionWithStatusCode). [here is commit](https://github.com/jejikeh/versity/pull/3/commits/857ef50539c99854c542971e8afd07fe7d4afe56#diff-e332d7f3a4bb1a268dc47c540de8ab60cb4cef6fef30c65a84b3beb3558e9ef8)

This is necessary in order not to write to users about problems on the service that they should not have known about. Well, that's how I see it)) 

But, for example, in the development build, all errors are visible and described in detail. 
[here's the whole file](https://github.com/jejikeh/versity/blob/857ef50539c99854c542971e8afd07fe7d4afe56/src/Versity.Users/External/Presentation/Controllers/ErrorController.cs)

Reply:
You will send it not to user. Client will handle it
But okey, I understand your point of view
But i don't catch idea of separate errors endpoint. For whom it? For me, as developer, it is more convinient to see error in response of server or even do not have handler of exceptions in dev enviroment(local). Developers are to lazy to do one more click to send request to this endpoint)
But this idea is interesting